### PR TITLE
drone_pushover_plugin added

### PR DESCRIPTION
**Pull request by awerv**

### CODE REVIEW

File: content/awerv/drone-pushover-plugin/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+version: '0.2'
```
Comment (tboerger) [inline content/awerv/drone-pushover-plugin/index.md:2]: This attribute got to be dropped.

File: content/awerv/drone-pushover-plugin/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+date: 2020-07-17T00:00:00+00:00
+title: Pushover
+author: awerv
+tags: [ Pushover, notifications ]
+logo: pushover.svg
+repo:  awerv/drone_pushover_plugin
```
Comment (tboerger) [inline content/awerv/drone-pushover-plugin/index.md:8]: Does it really use underscores in the name?

File: content/awerv/drone-pushover-plugin/index.md (hunk 1 above)
Comment (awerv) [reply content/awerv/drone-pushover-plugin/index.md]: Removed

File: content/awerv/drone-pushover-plugin/index.md (hunk 2 above)
Comment (awerv) [reply content/awerv/drone-pushover-plugin/index.md:7]: Yes, it does. Is it a big problem?

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Pull request closed without merging**